helpers/lib_tasks_docker.py

```diff
@@ -396,13 +396,13 @@ def docker_login(ctx, target_registry="aws_ecr.ck"):  # type: ignore
     """
     _ = ctx
     hlitauti.report_task()
-    # No login required as the `helpers` container is accessible on
-    # the public DockerHub registry.
+    # No login required as the `helpers` and `tutorials` images are accessible
+    # on the public DockerHub registry.
     if (
         not hserver.is_dev_ck()
-        and hrecouti.get_repo_config().get_name() == "//helpers"
+        and hrecouti.get_repo_config().get_name() in ["//helpers", "//tutorials"]
     ):
-        _LOG.warning("Skipping logging in for Helpers")
+        _LOG.warning("Skipping Docker login process for Helpers or Tutorials")
```

Comment:
I would remove the reference to helpers or tutorials at all

Reply:
Filed https://github.com/causify-ai/helpers/issues/424